x-pack/legacy/plugins/siem/public/components/timeline/body/column_headers/column_header.tsx

```diff
@@ -34,69 +32,74 @@ const ColumnHeaderComponent: React.FC<ColumneHeaderProps> = ({
   draggableIndex,
   header,
   timelineId,
+  isDragging,
   onColumnRemoved,
   onColumnResized,
   onColumnSorted,
   onFilterChange,
   sort,
 }) => {
-  const [isDragging, setIsDragging] = React.useState(false);
-  const handleResizeStop: ResizeCallback = (e, direction, ref, delta) => {
-    onColumnResized({ columnId: header.id, delta: delta.width });
+  const resizableEnable = { right: true };
```

Comment:
thanks for the refactoring here (and below) to ensure we're passing stable references to the `<Resizable>` 🙏

Reply:
with `useMemo` it can live inside, Thank you! :)